pyproject.toml

```
@@ -13,7 +13,7 @@ license = "Apache-2.0"
 dependencies = [
     "requests ~= 2.31",
     "boto3 >= 1.34.75",
-    "deadline >= 0.50,< 0.53",
```

If this is to bump the deadline version, we shouldn't pin to a patch version


```suggestion
    "deadline >= 0.50,< 0.54",
```

---

Fix in next rev